Makefile

```diff
@@ -169,6 +171,12 @@ test-conformance: $(GINKGO) $(KIND) $(SSM_PLUGIN) $(KUSTOMIZE) e2e-image ## Run
 
 test-conformance-fast: ## Run clusterctl based conformance test on workload cluster (requires Docker) using a subset of the conformance suite in parallel.
 	$(MAKE) test-conformance CONFORMANCE_E2E_ARGS="-kubetest.config-file=$(KUBETEST_FAST_CONF_PATH) -kubetest.ginkgo-nodes=5 $(E2E_ARGS)"
+
+# TODO(ashish-amarnath) what does "kubetest.ginkgo-nodes=5" do? is it required here?
+.PHONY: test-conformance-cni-antrea
+test-conformance-cni-antrea: $(GINKGO) $(KIND) $(SSM_PLUGIN) $(KUSTOMIZE) e2e-image ## Run network policy conformance tests on workload cluster using Antrea as CNI
```

Comment:
any thoughts on making this more generic, as described in https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/1745? cc @dims

Reply:
Yep.. looks like we have an example from the capz repo. I'll look into it.